manual/backflow_implementation.tex

```diff
@@ -30,31 +30,31 @@ \subsection{Value}
   ,WorkSpace.data(),Pivot.data(),PhaseValue);
 \end{lstlisting}
 
-QMCPACK represents the complex value of the wavefunction in polar coordinates $S=e^Ue^{i\theta}$. Specifically, \verb|LogValue| $U$ and \verb|PhaseValue| $\theta$ are handled separately. In the following, I will consider derivatives of the log value only.
+QMCPACK represents the complex value of the wavefunction in polar coordinates $S=e^Ue^{i\theta}$. Specifically, \verb|LogValue| $U$ and \verb|PhaseValue| $\theta$ are handled separately. In the following, we will consider derivatives of the log value only.
 
 \subsection{Gradient}
-To evaluate particle gradient of the log value of the Slater-Backflow wavefunction, we can use the $\log\det$ identity in eq.~\ref{eq:logdet}. This identity maps the derivative of $\log\det M$ with respect to a real variable $p$ to a trace over $M^{-1}dM$
+To evaluate particle gradient of the log value of the Slater-backflow wavefunction, we can use the $\log\det$ identity in Equation~\ref{eq:logdet}. This identity maps the derivative of $\log\det M$ with respect to a real variable $p$ to a trace over $M^{-1}dM$:
```

Comment:
May be it is more than necessary. I prefer brackets around equation numbers like (3).

Reply:
Equations references are currently without () throughout the manual.